S2VX.Game/Editor/NotesTimeline.cs

```diff
@@ -38,12 +57,14 @@ public class NotesTimeline : CompositeDrawable {
         };
 
         private const float TimelineHeight = 0.075f;
-        private const float TimelineWidth = 1.0f;
+        public const float TimelineWidth = 1.0f;
```

Comment:
I think I changed it to public to use it in SelectToolState but ended up not using it. I'll change it back and commit since it looks like everything works